dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -116,7 +116,7 @@ def check_docker_permission(verbose) -> bool:
 
 
 def compare_version(current_version: str, min_version: str) -> bool:
-    return version.parse(current_version) > version.parse(min_version)
+    return version.parse(current_version) >= version.parse(min_version)
```

Comment:
This is the first error I found (the 20.10.0 version was actually failing the check :scream: )